Selenium_cource/Homework1_2/Change_password.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'Verification'
+
+module Change_password
```

Comment:
I wouldn't  create a module if it will contain only one method
It makes project structure complicated and not clear